cmd/kubelet/app/options/options.go

```diff
@@ -59,7 +59,7 @@ func NewKubeletServer() *KubeletServer {
 
 // AddFlags adds flags for a specific KubeletServer to the specified FlagSet
 func (s *KubeletServer) AddFlags(fs *pflag.FlagSet) {
-	fs.StringVar(&s.Config, "config", s.Config, "Path to the config file or directory of files")
+	fs.StringVar(&s.PodManifestPath, "config", s.PodManifestPath, "Path to the config file or directory of files")
```

Comment:
Can we deprecate this flag and instead add a new flag called `--pod-manifest-path` while we are at it?